app/services/AuditService.scala

```diff
@@ -207,6 +214,23 @@ class AuditService @Inject() (auditConnector: AuditConnector)(implicit ec: Execu
     )
   }
 
+  private def toDdinProgressAuditDetail(journey: Journey, continueOrExitBool: Boolean)
+    (implicit r: AuthenticatedJourneyRequest[_]): DdInProgressAuditDetail = {
+    val continueOrExit: String = continueOrExitBool match {
```

Comment:
it'd probably be a cleaner to just to `if(continueOrExitBool) "continue" else "exit"` since it's just a boolean. Also, it's nice when the name of the boolean tells you what true or false means to avoid ambiguity - maybe something like `hasChosenToContinue` would convey that?